manager/types/dispatcher.go

```diff
@@ -19,25 +20,30 @@ type ReqMsg struct {
 	Options      map[string]interface{} `json:"options"`
 }
 
+// UpdateState is to uptade the state of pluginpb.
 func (r *ReqMsg) UpdateState(stat pluginpb.STATE) {
 	r.State = stat
 }
 
+// UpdateSeverity is to uptade the severity of pluginpb.
 func (r *ReqMsg) UpdateSeverity(sev pluginpb.SEVERITY) {
 	r.Severity = sev
 }
 
+// UpdateMSG is to update message
 func (r *ReqMsg) UpdateMSG(message string) {
 	r.Msg = message
 }
 
+// DiscordMsg is type for sending messages to a discord.
 type DiscordMsg struct {
 	Username  string  `json:"username,omitempty"`
 	AvatarURL string  `json:"avatar_url,omitempty"`
 	Content   string  `json:"content,omitempty"`
 	Embeds    []Embed `json:"embeds"`
 }
 
+// Embed is imformation for detail message.
```

Comment:
@kim201212 Please fixed typo

Reply:
Fix the typo
https://github.com/dsrvlabs/vatz/pull/461/commits/3ee0c0617c951831cd45c64b7780d9cc644ba416